docs/presigned_urls.md

```diff
@@ -0,0 +1,354 @@
+```
+aws s3 presign s3://EXAMPLE_BUCKET/EXAMPLE_OBJECT --expires-in 604800
+```
+Example output:
+```
+https://EXAMPLE_BUCKET.s3.us-east-1.amazonaws.com/myobject?AWSAccessKeyId=AKIAEXAMPLEXXX&Expires=604800&Signature=ibOGfAovnhASUASsdasjj321dtg2s%3D
+```
+?> Note: When presigned URLs are created using the security credentials of an IAM user, the URL can last up to 7 days. When using a temporary token, the URL expires when the token expires, even if the specified time duration is longer.
+
+For further optional arguments of this AWS CLI function, see the detailed documentation [here](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/presign.html).
+
+One can also use AWS SDKs like *boto3* for Python to generate presigned URLs, using the `generate_presigned_url` function, using the [following script](../scripts/presigned_url_scripts/create_presigned_urls.py):
+```python
+import boto3
+from botocore.exceptions import ClientError
+
+def generate_presigned_URL(bucket_name, object_path, expiration=3600):
+    """
+    Generates a presigned URL that can be used to share a private S3 object.
+
+    :param bucket_name: The name of S3 bucket containing private objects.
+    :type bucket_name: string
+    :param object_path: The relative path of S3 object to be shared via presigned URL.
+    :type object_path: string
+    :param expiration: The number of seconds the presigned URL is valid for.
+    :type expiration: int
+    :return: The presigned URL.
+    :rtype: string or None
+    """
+
+    # Create low-level S3 service client
+    s3_client = boto3.client("s3")
+
+    try:
+        # URL generated by AWS user with access to private object (i.e. the owner)
+        url = s3_client.generate_presigned_url(
+            ClientMethod="get_object", # HTTP Method
+            Params={
+                "Bucket": bucket_name, # S3 bucket
+                "Key": object_path # relative object path
+            },
+            ExpiresIn=expiration # URL duration in seconds
+        )
+    except ClientError as err:
+        print(err)
+        logging.error(err)
+        return None
+
+    # Return the presigned URL as a string
+    return url
+```
+Further information on this *boto3* function can be found [here](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/s3-presigned-urls.html).
+
+Non-programmatically, a presigned URL can be generated using the S3 console or AWS Explorer for Visual Studio – [comprehensive documentation on presigned URLs can be found here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-presigned-url.html). Once a presigned URL is created for a private object, it can be used within Chromoscope configuration files to temporarily visualize these private files.
+
+&nbsp;
+## Configuration File Creation Using Presigned URLs
```

Comment:
Could be shortened to make it more readable on the sidebar

<img width="266" alt="Screenshot 2023-04-24 at 17 25 16" src="https://user-images.githubusercontent.com/9922882/234120287-9ba0b433-e112-4909-8e43-0ae39be14e3f.png">